rcongui/src/components/LogsView/logs.js

```diff
@@ -158,6 +212,17 @@ class Logs extends React.Component {
               <IconButton onClick={onFullScreen}>
                 {isFullScreen ? <FullscreenExitIcon /> : <FullscreenIcon />}
               </IconButton>
+            <FormControlLabel
```

Comment:
I'm not quite sure if we want to have this in the main UI tbh. It is a setting which just makes the whole UI more stuffed with things. What about moving it to the settings page (next to the theme switcher or something)?

Reply:
I don't have the patience to figure out how to move the switch to the other page. :(

I don't really like it here either, but it is only taking up existing whitespace. There really isn't any other frontend-UI setting other than the theme setting, which is global rather than for a particular widget so I don't see any existing pattern to follow.

And the switch is very discoverable when it is right on the page. :)

![image](https://github.com/MarechJ/hll_rcon_tool/assets/2712638/9a61b72c-2c7b-4282-aa9d-6a7caf212fdd)

Reply:
Ok, then let's leave it there for now. There should be a better, more thought-through UI experience at some point, anyway :D